lib/src/sidenav/Sidenav.tsx

```diff
@@ -1,56 +1,140 @@
+  <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="currentColor">
+    <path d="M0 0h24v24H0z" fill="none" />
+    <path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z" />
+  </svg>
+);
 
+const collapsedIcon = (
+  <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="currentColor">
+    <path d="M0 0h24v24H0z" fill="none" />
+    <path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z" />
+  </svg>
+);
+
+const externalLinkIcon = (
+  <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24" fill="currentColor">
+    <path d="M0 0h24v24H0z" fill="none" />
+    <path d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z" />
+  </svg>
+);
+
+const DxcSidenav = ({ children, title }: SidenavPropsType): JSX.Element => {
+  const colorsTheme = useTheme();
   return (
     <ThemeProvider theme={colorsTheme.sidenav}>
-      <SidenavContainer padding={padding}>
-        <BackgroundColorProvider color={colorsTheme.sidenav.backgroundColor}>{children}</BackgroundColorProvider>
+      <SidenavContainer>
+        <BackgroundColorProvider color={colorsTheme.sidenav.backgroundColor}>
+          {title}
+          <DxcStack divider={true} gutter="small">
+            {children}
+          </DxcStack>
+        </BackgroundColorProvider>
       </SidenavContainer>
     </ThemeProvider>
   );
 };
 
-const Title = ({ children }: SidenavTitlePropsType): JSX.Element => <SidenavMenuTitle>{children}</SidenavMenuTitle>;
-
-const Subtitle = ({ children }: SidenavSubtitlePropsType): JSX.Element => (
-  <SidenavMenuSubTitle>{children}</SidenavMenuSubTitle>
+const Title = ({ children, icon }: SidenavTitlePropsType): JSX.Element => (
+  <SidenavTitle>
+    {typeof icon === "string" ? <img src={icon} /> : icon}
+    {children}
+  </SidenavTitle>
 );
 
-const Link = ({ tabIndex = 0, href, onClick, children }: SidenavLinkPropsType): JSX.Element => {
-  const setIsSidenavVisibleResponsive = useResponsiveSidenavVisibility();
-  const handleClick = () => {
-    onClick?.();
-    setIsSidenavVisibleResponsive?.(false);
-  };
+const Section = ({ children }: SidenavSectionPropsType): JSX.Element => <DxcStack>{children}</DxcStack>;
 
+const Group = ({ children, title, collapsable = false, icon }: SidenavGroupPropsType): JSX.Element => {
+  const [collapsed, setCollapsed] = useState(false);
   return (
-    <SidenavMenuLink tabIndex={tabIndex} href={href} onClick={handleClick}>
-      {children}
-    </SidenavMenuLink>
+    <SidenavGroup>
+      {collapsable && title ? (
+        <SidenavGroupTitleButton aria-expanded={collapsed} onClick={() => setCollapsed(!collapsed)}>
+          <SidenavSpan>
+            {typeof icon === "string" ? <SidenavIconImg src={icon} /> : icon}
+            {title}
+          </SidenavSpan>
+          {collapsed ? collapsedIcon : collapsableIcon}
```

Comment:
Swipe these icons. Also, they are currently bigger than what Alex thought for them:

![image](https://user-images.githubusercontent.com/44321109/176679347-1de415b4-11c2-4d79-a327-d5fa871c6877.png)

Alex design: 

![image](https://user-images.githubusercontent.com/44321109/176679496-bac8a907-6ced-445d-be9f-6b809869ea3b.png)

Reply:
Yes I also saw this issue and I asked him directly regarding this. And having it as 16x16 it was too small so it was increased to 24x24.

Reply:
I keep thinking that 24x24 is too big, maybe we can reach an intermediate solution while trying to preserve the whole size of 24x24:

<img width="167" alt="image" src="https://user-images.githubusercontent.com/44321109/176852459-03edc351-d5ed-48b9-a234-29a7c40cf06a.png">

const SidenavCollapseIcon = styled.span`
  display: flex;
  flex-wrap: wrap;
  align-content: center;
  padding: 3px;
  height: 18px;
  width: 18px;
`;

(Forget the margin of the image)